src/navigation/nav2_simple_navigator/src/main.cpp

```diff
@@ -13,14 +13,19 @@
 // limitations under the License.
 
 #include <memory>
+#include <stdexcept>
 #include "rclcpp/rclcpp.hpp"
 #include "nav2_simple_navigator/simple_navigator.hpp"
 
 int main(int argc, char ** argv)
 {
+  try {
```

Comment:
I don't want to diverge this file from all of the other main.cpp's. If we want to catch exceptions at the top, we should do it for all of them. Let's discuss as a group.